tools/BatteryModel.py

```diff
@@ -0,0 +1,42 @@
+# Class model of a standard LiPo battery
+
+import sys
+import pyulog as ulog
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import curve_fit
+
+class BatteryModel:
+    def __init__(self, file, nominal_voltage: float = 11.1, nominal_mAh: float = 5100.0, cells: int = 3):
+   
+        # Givens
+        # self.__nominal_voltage = nominal_voltage
+        self.__cells = cells                      
+        self.__nominal_Ah = nominal_mAh/1000.0
+        
+        self.__nominal_capacity = 8500      # mAh
+
+    
+    # Scale capacity estimation based on external temperature
+    def __scale_temperature(self):
+        pass
+
+    def __drain_curve_percent(self, x):
+        return 58.821010559671066*x**3 + -2134.5758966115527*x**2 + 25869.660996405204*x + -104622.36370746762
```

Comment:
> NOTE: hardcoded values (magic numbers) like this are bad practice. It's okay for this revision but just wanted to make you guys aware.

If you _need_ to used magic numbers, always add a comment. E.g. for this case, write something like:

```py
def __drain_curve_percent(self, x):
        # This polynomial was generated by <script_name>.py and represents the
        # relationship between "x" and "y"
        return 58.821010559671066*x**3 + -2134.5758966115527*x**2 + 25869.660996405204*x + -104622.36370746762
```